config/backpack/crud.php

```diff
@@ -162,444 +162,12 @@
     |-------------------
     */
 
-    'show_translatable_field_icon'     => true,
+    'show_translatable_field_icon'     => 'true',
```

Comment:
Does `true` behave the same way as `'true'` for backpack?
```suggestion
    'show_translatable_field_icon'     => true,
```